lib/main.js

```diff
@@ -205,6 +208,7 @@ exports.main = function(options, callbacks) {
       }
     }
     noatb.install();
+    oninstall.install(is_install);
```

Comment:
@bsstoner There is the obvious benefit that the extension will not inject the script unnecessarily. I am not sure why I did not come up with this myself. Will change this too.